tests/integration_tests/functional/test_logging.py

```diff
@@ -245,6 +245,24 @@ def test_api_requests_logs(test_microvm_with_api):
     assert log_file_contains_strings(log_fifo, expected_log_strings)
 
 
+def test_cmd_line_logger_config(test_microvm_with_api):
+    """Test logger configuration with fifo parameter."""
+    microvm = test_microvm_with_api
+
+    log_fifo_path = os.path.join(microvm.path, 'log_fifo')
+    log_fifo = log_tools.Fifo(log_fifo_path)
+    microvm.create_jailed_resource(log_fifo.path, create_jail=True)
+
+    microvm.jailer.extra_args = {'log-path': 'log_fifo'}
+    microvm.spawn()
+    microvm.basic_config()
+    microvm.start()
```

Comment:
Is it necessary to start the microVM for this test? The "Running Firecracker" message should be received as fast the logger init was called right?

Reply:
No, it's not. I think I wanted to make sure that we can also send API requests to the socket, but this test shouldn't, however, check this. I removed it. :+1: